include/puara/descriptors/touch.h

```diff
@@ -109,11 +93,11 @@ class Touch
           //       (std::abs(movement * 0.15)), multiRub[i], 0.7, leakyRubFreq,
           //       leakyRubTimer);
           //
-          multiBrush[i] = multiBrushIntegrator[i].integrate(movement * 0.15);
-          multiRub[i] = multiRubIntegrator[i].integrate(std::abs(movement * 0.15));
+          multiBrush[i] = multiBrushIntegrator[i].integrate(movement[i] * 0.15);
+          multiRub[i] = multiRubIntegrator[i].integrate(std::abs(movement[i] * 0.15));
```

Comment:
Here we can promote brush and rub as specific classes (separate from the less interesting gestures from this class)